contracts/common/AuthorizedMinterControl.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.8.20;
+
+/**
+ * @title AuthorizedMinterControl
+ * @dev Abstract contract to manage authorized minters for ERC1155M tokens
+ */
+abstract contract AuthorizedMinterControl {
```

Comment:
i think we should eventually add Ownable to this, but we have mixed versions of Ownable in the codebase so it will break things. I've opted to make the addAuthorizedMinter functions virtual so the implementing contract can add its own access control